src/index.js

```diff
@@ -46,6 +46,14 @@ class ServerlessReqValidatorPlugin {
 
     this._beforeDeploy = this.beforeDeploy.bind(this)
 
+    // Create schema for your properties. For reference use https://github.com/ajv-validator/ajv
+    serverless.configSchemaHandler.defineFunctionEventProperties('aws', 'http', {
```

Comment:
With the 1.0.4 release I'm now getting this error when starting our application:

```TypeError: Cannot read property 'defineFunctionEventProperties' of undefined```

Is there a dependent package that needs to be added?

Reply:
What framework version are your running ?

Reply:
Hi, thanks for the response. Our repo (which is private), is actually locked to severless 1.74 due to another issue. I can revisit that and determine if that still exists.